src/App/App.js

```diff
@@ -19,13 +27,49 @@ injectGlobal`
   html,
   body {
     margin: 0;
+    font-size: 20px;
   }
   html,
   body,
   #root {
     width: 100%;
     height: 100%;
   }
+
+  @font-face {
+    font-family: OCRAStd;
+    src: url('${OCRAStdRegular}') format('opentype');
+    font-weight: 400;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: RobotoMono;
+    src: url('${RobotoMonoLight}') format('opentype');
+    font-weight: 300;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: RobotoMono;
+    src: url('${RobotoMonoRegular}') format('opentype');
+    font-weight: 400;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: RobotoMono;
+    src: url('${RobotoMonoMedium}') format('opentype');
+    font-weight: 500;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: RobotoMono;
+    src: url('${RobotoMonoBold}') format('opentype');
+    font-weight: 700;
+    font-style: normal;
+  }
```

Comment:
Is it a good idea to import the CSS rules from another file? Just so `App.js ` doesn't get to be just a very long css template literal.

Reply:
@Comum agree. I've moved `injectGlobal` to a `global-styles.js` file

Reply:
👍